From 1fcca50970ef9894b395d55f0deeb57bfbc3ed7b Mon Sep 17 00:00:00 2001
From: Paul Naoumenko <paulx.naoumenko@intel.com>
Date: Wed, 5 Dec 2012 11:33:25 +0100
Subject: [PORT FROM MAIN] wifi: [JB MR1 Bring Up] REVERT ME: temporarily relocate nvs.bin file

BZ: 70883

Calibration can not write in /factory partition

Change-Id: Ifc6949f46ba32a7db2e074a2bd4006e38fec57cd
Orig-Change-Id: I19f456e0ca461b03611a27040f0fb588302de573
Signed-off-by: Paul Naoumenko <paulx.naoumenko@intel.com>
Reviewed-on: http://android.intel.com:8080/79148
Reviewed-by: Bachot, Jean-michel <jean-michel.bachot@intel.com>
Reviewed-by: Trivelly, Jean <jean.trivelly@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 wlan_prov/Android.mk          |    2 +-
 wlan_prov/wlan_provisioning.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/wlan_prov/Android.mk b/wlan_prov/Android.mk
index 3b1e09a..79f7695 100644
--- a/wlan_prov/Android.mk
+++ b/wlan_prov/Android.mk
@@ -36,7 +36,7 @@ $(SYMLINKS): $(LOCAL_INSTALLED_MODULE) $(LOCAL_PATH)/Android.mk
 	@echo "Symlink: $@"
 	@mkdir -p $(dir $@)
 	@rm -rf $@
-	$(hide) ln -sf $(addprefix /factory/wifi/,$(NVS_FILE)) $@
+	$(hide) ln -sf $(addprefix /data/misc/wifi/,$(NVS_FILE)) $@
 
 ALL_DEFAULT_INSTALLED_MODULES += $(SYMLINKS)
 ALL_MODULES.$(LOCAL_MODULE).INSTALLED := \
diff --git a/wlan_prov/wlan_provisioning.c b/wlan_prov/wlan_provisioning.c
index c5a153d..1df7dfd 100644
--- a/wlan_prov/wlan_provisioning.c
+++ b/wlan_prov/wlan_provisioning.c
@@ -37,7 +37,7 @@
 const unsigned char NullMacAddr[MAC_ADDRESS_LEN] = { 0, 0, 0, 0, 0, 0 };
 
 #define WIFI_PATH "/data/misc/wifi"
-const char NVS_file_name[] = "/factory/wifi/wl1271-nvs.bin";
+const char NVS_file_name[] = "/data/misc/wifi/wl1271-nvs.bin";
 const char Default_NVS_file_name[] = "/system/etc/wifi/wl1271-nvs.bin";
 const char WLAN_SDIO_BUS_PATH[] = "/sys/bus/sdio/drivers/wl1271_sdio/";
 #define WLAN_DRV_SDIO_NAME "wl1271_sdio"
-- 
1.7.4.1

